test/algorithms/buffer/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ endforeach()
 foreach(item IN ITEMS
     buffer_point_geo
     buffer_polygon_geo
+    buffer_linestring
+    buffer_linestring_geo
```

Comment:
For this case I had to exclude 5 cases.

I will investigate them later this year, after integrating some other enhancements (robust side, #1404, ...)